als-server/shared/src/test/scala/org/mulesoft/als/server/modules/diagnostic/support/avro/AvroSupportTest.scala

```diff
@@ -0,0 +1,65 @@
+package org.mulesoft.als.server.modules.diagnostic.support.avro
+
+import amf.core.client.common.validation.ProfileNames
+import org.mulesoft.als.server.modules.diagnostic.BasicCleanDiagnosticTest
+import org.mulesoft.lsp.feature.diagnostic.DiagnosticSeverity
+
+
+class AvroSupportTest extends BasicCleanDiagnosticTest {
+
+  test("Async importing valid Avro") {
+    withServer(buildServer()) { server =>
+      for {
+        d <- requestCleanDiagnostic(server)(filePath("avro/async26-imports-valid-avro.yaml"))
+      } yield {
+        server.shutdown()
+        d.foreach(filediag => assert(filediag.diagnostics.isEmpty))
+        assert(d.size == 2)
+        assert(d.head.profile == ProfileNames.ASYNC26)
+      }
+    }
+  }
+
+  test("Async importing invalid Avro") {
+    withServer(buildServer()) { server =>
+      for {
+        d <- requestCleanDiagnostic(server)(filePath("avro/async26-imports-invalid-avro.yaml"))
+      } yield {
+        server.shutdown()
+        assert(d.size == 2)
+        assert(d.head.diagnostics.length == 1)
+        val errorMessage = d.head.diagnostics.head.message.toLowerCase()
+        assert(errorMessage.contains("duplicate") && errorMessage.contains("address"))
+        assert(d.head.diagnostics.head.severity.get == DiagnosticSeverity.Error)
+        assert(d.head.profile == ProfileNames.ASYNC26)
+      }
+    }
+  }
+
+
+  test("avro containing inline avro") {
```

Comment:
not sure what `inline avro` inside an avro means, maybe just leave this as "valid complex avro" or something?